sdk/servicebus/azure-servicebus/azure/servicebus/amqp/_amqp_message.py

```diff
@@ -141,19 +159,16 @@ def __init__(
                 "or value_body being set as the body of the AmqpAnnotatedMessage."
             )
 
-        self._body = None
-        self._body_type = None
         if "data_body" in kwargs:
-            self._body = kwargs.get("data_body")
-            self._body_type = uamqp.MessageBodyType.Data
+            self._data_body = normalized_data_body(kwargs.get("data_body"))
+            self.body_type = AmqpMessageBodyType.DATA
```

Comment:
wondering why we want to allow users to set body_type as well now? are there cases where we would want to allow users to set this?

Reply:
I think this is probably my mistake.... will fix.